README.md

```diff
@@ -94,138 +73,103 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Authentication, User can manage his categories and transactions ]**
+- **[User must be authenticated]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- Project Documentation -->
+## 📽️ Project Doocumentation <a name="pro-doc"></a>
+
+Here is a link to the [Video Link](https://www.loom.com/share/7a069d3c9ba841f98177f4365011a177) of the project.
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Budget-App Coming soon...](https://budget-app.herokuapp.com/)
```

Comment:
- [ ] when trying to open the live demo it is only displaying the word "Hi!" please make sure to deploy the project.

Reply:
Hi Mr. @mikethreels,
This link is not working I have to try to deploy this project but I am having some issues. for deployment the need to first add a payment method. when I add the details of my card then the card is declined and never saved. I will try to fix it later.